packages/app/obojobo-document-engine/src/scripts/oboeditor/components/code-editor.js

```diff
@@ -1,15 +1,9 @@
-import React from 'react'
+import React, { Suspense } from 'react'
```

Comment:
A birdie told me Suspense was experimental, are we okay using it?

Reply:
Oh dang, good call.  Not sure what experimental means here - is it just not in the production dist?  is it just that the api will change a lot?  is it that it's risky and buggy?

Reply:
https://reactjs.org/blog/2018/11/27/react-16-roadmap.html  Looks like it's been in since 16.6.0, we're using 16.13.1.  I think it's also experimental due to server side rendering support not being done yet.

> Recommendation: If you only do client rendering, we recommend widely adopting React.lazy() and <React.Suspense> for code splitting React components. If you do server rendering, you’ll have to wait with adoption until the new server renderer is ready.

We're not using it for server rendering here, so I say it's a go.